src/Pingvinas.Event.Domain/Repositories/EventRepository.cs

```diff
@@ -49,15 +49,15 @@ private PingvinEvent CreatePingvinEvent()
         var eventId = Guid.NewGuid().ToString();
         var commerceData = new Commerce();
         var nameData = new Name();
+        var internetData = new Internet();
         return new PingvinEvent
         {
             CreatorId = Guid.NewGuid().ToString(),
-            Creator = new User
-            {
-                Name = nameData.FullName(),
-                Active = true,
-                Id = Guid.NewGuid().ToString()
-            },
+            Creator = new User(
```

Comment:
Ikke så veldig fan av denne endringen. 🤔
noen argumenter for å ikke gjøre det sånn som det var? 😅
```
{
    Name = nameData.FullName(),
    Active = true,
    Id = Guid.NewGuid().ToString()
}
```

Reply:
Siden det nå er `record` så må det nesten være slik?

Er det bruk av constructor eller å kreve mailadressen du tenker på?

Reply:
Ah ja det er record ja, men hva med dette? 😊
```
Creator = new User(
    Id: Guid.NewGuid().ToString(),
    Name: nameData.FullName(),
    Email: internetData.Email(),
    Active: true),
```

Reply:
Sure!

Reply:
Hmmmmmmmmmmm
nei glem det, det er litt unødvendig.. 😅
Har bare reviewet dette på github, i en IDE så er det ikke noe issue.

Reply:
Hehe, ok